packages/neuvector/generated-changes/patch/Chart.yaml.patch

```diff
@@ -0,0 +1,22 @@
+--- charts-original/Chart.yaml
++++ charts/Chart.yaml
+@@ -1,11 +1,17 @@
++annotations:
++  catalog.cattle.io/certified: partner
++  catalog.cattle.io/display-name: NeuVector
++  atalog.cattle.io/release-name: neuvector
```

Comment:
typo here, should be `catalog`

Reply:
Neuvector helm chart configures docker as the container runtime by default. Please change the vaule as per your cluster configuration. I will resubmit the PR by fixing typo error.

please check the cluster runtime
kubectl get node -o json | jq -r '.items[0].status.nodeInfo.containerRuntimeVersion' | awk -F':' '{print $1}'

Modify below values as per above output
docker:
  path: /var/run/docker.sock
  enabled: false

containerd:
  enabled: false
  path: /var/run/containerd/containerd.sock

crio:
  enabled: false
  path: /var/run/crio/crio.sock